TestSwitchApi/TestSwitchApi/DataMigrations/TestSwitchScript.sql

```diff
@@ -0,0 +1,215 @@
+
```

Comment:
Can we make the name of this file a bit more descriptive? Something like `InitialSchemaSetup.sql`?

Reply:
File renamed to InitialSchemaSetup.sql